lib/jsdom.js

```diff
@@ -185,13 +194,28 @@ exports.env = exports.jsdom.env = function () {
       config.html = responseText;
       config.url = res.request.uri.href;
 
+      var contentType = res.headers["content-type"];
+
       if (config.parsingMode === "auto" && (
-        res.headers["content-type"] === "application/xml" ||
-        res.headers["content-type"] === "text/xml" ||
-        res.headers["content-type"] === "application/xhtml+xml")) {
+        contentType === "application/xml" ||
+        contentType === "text/xml" ||
+        contentType === "application/xhtml+xml")) {
         config.parsingMode = "xml";
       }
 
+      var cookies = res.headers["set-cookie"];
+
+      if(cookies) {
```

Comment:
Really missing things like this in linter/[editor config](http://editorconfig.org/).